spec/Overview-WebCryptoAPI.xml

```diff
@@ -14617,25 +14615,25 @@ required <a href="#dfn-HashAlgorithmIdentifier">HashAlgorithmIdentifier</a> <dfn
               <tr>
                 <td>Get key length</td>
                 <td>None</td>
-                <td>Integer or null</td>
+                <td>null</td>
               </tr>
             </tbody>
           </table>
         </div>
-        <div id="hkdf-ctr-params" class="section">
-          <h4>HkdfCtrParams dictionary</h4>
+        <div id="hkdf-params" class="section">
+          <h4>HkdfParams dictionary</h4>
           <x:codeblock language="idl">
-dictionary <dfn id="dfn-HkdfCtrParams">HkdfCtrParams</dfn> : <a href="#dfn-Algorithm">Algorithm</a> {
-<span class="comment">// The algorithm to use with HMAC (e.g.: <a href="#alg-sha-256">SHA-256</a>)</span>
-required <a href="#dfn-HashAlgorithmIdentifier">HashAlgorithmIdentifier</a> <dfn id="dfn-HkdfCtrParams-hash">hash</dfn>;
-<span class="comment">// A bit string that corresponds to the label that identifies the purpose for the derived keying material.</span>
-required BufferSource <dfn id="dfn-HkdfCtrParams-label">label</dfn>;
-<span class="comment">// A bit string that corresponds to the context of the key derivation, as described in Section 5 of [<a href="#SP800-108">NIST SP800-108</a>]</span>
-required BufferSource <dfn id="dfn-HkdfCtrParams-context">context</dfn>;
+dictionary <dfn id="dfn-HkdfParams">HkdfParams</dfn> : <a href="#dfn-Algorithm">Algorithm</a> {
+  <span class="comment">// The algorithm to use with HMAC (e.g.: <a href="#alg-sha-256">SHA-256</a>)</span>
+  required <a href="#dfn-HashAlgorithmIdentifier">HashAlgorithmIdentifier</a> <dfn id="dfn-HkdfParams-hash">hash</dfn>;
+  <span class="salt">// A bit string that corresponds to the salt used in the extract step.</span>
```

Comment:
This should remain `class="comment"` - it causes the line to appear as a comment in the code block